web/packages/plugins/shared/common/src/components/Pickers/EntityPicker.tsx

```diff
@@ -1,94 +1,43 @@
 }) => {
   // TODO: Valid types should be passed to this, and filtered for in the view
-  const { onChange, formData } = props
+  const { onChange, formData, blueprintFilter = BlueprintEnum.PACKAGE } = props
   const [showModal, setShowModal] = useState<boolean>(false)
-  // @ts-ignore-line
-  const { token } = useContext(AuthContext)
-  const dmssAPI = new DmssAPI(token)
-  const dataSources: IDataSources = useDataSources(dmssAPI)
-  const application = useContext(ApplicationContext)
-
-  const onSelect = (nodeData: TreeNode) => {
-    setShowModal(false)
-    onChange({
-      name: nodeData.name || '',
-      type: nodeData.type,
-      _id: nodeData.nodeId,
-    })
-  }
 
   return (
     <div style={{ display: 'flex', flexDirection: 'row' }}>
-      {/*<Input*/}
-      {/*  type="string"*/}
-      {/*  value={formData?.name || formData?._id || ''}*/}
-      {/*  placeholder="Select"*/}
-      {/*  onClick={() => setShowModal(true)}*/}
-      {/*  style={{ width: '280px', margin: '0 8px', cursor: 'pointer' }}*/}
-      {/*/>*/}
-      {/*<Modal*/}
-      {/*  toggle={() => setShowModal(!showModal)}*/}
-      {/*  open={showModal}*/}
-      {/*  title={'Select an Entity'}*/}
-      {/*>*/}
-      {/*  <Tree*/}
-      {/*    state={index.models.tree.models.tree}*/}
-      {/*    operations={index.models.tree.operations}*/}
-      {/*  >*/}
-      {/*    {(renderProps: TreeNodeRenderProps) => {*/}
-      {/*      const { nodeData } = renderProps*/}
-
-      {/*      if (nodeData.meta.type !== BlueprintEnum.PACKAGE) {*/}
-      {/*        const onClick = () => {*/}
-      {/*          onSelect(nodeData)*/}
-      {/*        }*/}
-      {/*        return (*/}
-      {/*          <div*/}
-      {/*            style={{ display: 'flex', flexDirection: 'row' }}*/}
-      {/*            onClick={onClick}*/}
-      {/*          >*/}
-      {/*            {renderProps.iconGroup(() => onClick)}*/}
-      {/*            {nodeData.title}*/}
-      {/*          </div>*/}
-      {/*        )*/}
-      {/*      } else {*/}
-      {/*        return (*/}
-      {/*          <div*/}
-      {/*            onClick={() => handleOpenOrExpand(renderProps)}*/}
-      {/*            style={{*/}
-      {/*              display: 'flex',*/}
-      {/*            }}*/}
-      {/*          >*/}
-      {/*            {renderProps.iconGroup(() => handleOpenOrExpand(renderProps))}*/}
-      {/*            {nodeData.title}*/}
-      {/*            {nodeData.isLoading && (*/}
-      {/*              <small style={{ paddingLeft: '15px' }}>Loading...</small>*/}
-      {/*            )}*/}
-      {/*          </div>*/}
-      {/*        )*/}
-      {/*      }*/}
-      {/*    }}*/}
-      {/*  </Tree>*/}
-      {/*</Modal>*/}
+      <Input
+        type="string"
+        value={formData?.name || formData?._id || ''}
+        placeholder="Select"
+        onClick={() => setShowModal(true)}
+        style={{ width: '280px', margin: '0 8px', cursor: 'pointer' }}
```

Comment:
tips about styling:

for divs / popups / inputs, using the vw for width and vh for height is often better than to hardcode pixel values for width and height, since this will scale the size relative to the browser window size

https://alligator.io/css/viewport-units/

Reply:
done